From 0053e7c332a7439a0958e3740fdb0e407826eba4 Mon Sep 17 00:00:00 2001
From: Vincent Abriou <vincent.abriou@st.com>
Date: Wed, 10 Apr 2019 16:07:18 +0200
Subject: [PATCH 1/1] TFLite pip package: fix _interpreter_wrapper.so undefined
 symbol

Build dependencies of interpreter_wrapper such as:
numpy.cc
python_error_reporter.cc
python_utils.cc

Signed-off-by: Vincent Abriou <vincent.abriou@st.com>
---
 tensorflow/lite/tools/pip_package/setup.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tensorflow/lite/tools/pip_package/setup.py b/tensorflow/lite/tools/pip_package/setup.py
index 38d3fb3..cc060c4 100644
--- a/tensorflow/lite/tools/pip_package/setup.py
+++ b/tensorflow/lite/tools/pip_package/setup.py
@@ -143,7 +143,10 @@ ext = Extension(
     name='%s._interpreter_wrapper' % PACKAGE,
     language='c++',
     sources=['interpreter_wrapper/interpreter_wrapper.i',
-             'interpreter_wrapper/interpreter_wrapper.cc'],
+             'interpreter_wrapper/interpreter_wrapper.cc',
+             'interpreter_wrapper/numpy.cc',
+             'interpreter_wrapper/python_error_reporter.cc',
+             'interpreter_wrapper/python_utils.cc'],
     swig_opts=['-c++',
                '-I%s' % TENSORFLOW_DIR,
                '-module', 'interpreter_wrapper',
-- 
2.7.4

